tests/testthat/test-tidied_df.R

```diff
@@ -0,0 +1,48 @@
+# standard clean dataset
+data <- list(year = c(2000, 2001, 2002, 2003, 2004, 2005),
```

Comment:
All of these appear to be lists, but normally we'd pass the function a data frame wouldn't we?